main.tf

```diff
@@ -146,6 +146,7 @@ resource "aws_cloudfront_distribution" "cloudfront_distribution" {
       cached_methods   = cache_behavior.value.cached_methods
       target_origin_id = cache_behavior.value.target_origin_id
       compress         = lookup(cache_behavior.value, "compress", null)
+      trusted_signers  = cache_behavior.value.trusted_signers
```

Comment:
This should be optional as well:
`lookup(ache_behavior.value, "trusted_signers", null)`

In addition this should be added to the default cache behaviors as well